lightwood/mixers/sklearn_mixer.py

```diff
@@ -0,0 +1,185 @@
+import numpy as np
+
+from sklearn.svm import SVC
+from sklearn.linear_model import SGDClassifier, SGDRegressor, LinearRegression
+from sklearn.naive_bayes import GaussianNB
+from sklearn.metrics import r2_score, balanced_accuracy_score
+
+from lightwood.logger import log
+from lightwood.constants.lightwood import COLUMN_DATA_TYPES
+from lightwood.mixers import BaseMixer
+
+CLASSIFICATION_MODELS = [
+    (SGDClassifier, {}),
+    (SVC, {}),
+    (GaussianNB, {}),
+]
+
+REGRESSION_MODELS = [
+    (LinearRegression, {}),
+    (SGDRegressor, {}),
+]
+
+class SklearnMixer(BaseMixer):
+    def __init__(self):
+        super().__init__()
+        self.targets = None
+        self.transformer = None
+        self.encoders = None
+
+    def fit(self, train_ds, test_ds=None):
+        """
+        :param train_ds: DataSource
+        :param test_ds: DataSource
+        """
+        self.fit_data_source(train_ds)
+
+        self.targets = {}
+
+        if test_ds is None:
+            test_ds = train_ds.subset(0.1)
+        else:
+            self.fit_data_source(test_ds)
+            assert test_ds.transformer is train_ds.transformer
+            assert test_ds.encoders is train_ds.encoders
```

Comment:
Again, why assert here and not as part of the tests ?

Reply:
https://github.com/mindsdb/lightwood/pull/277#discussion_r495425909